Dockerfile.plugins

```diff
@@ -1,7 +1,8 @@
-FROM public.ecr.aws/amazonlinux/amazonlinux:latest
+FROM public.ecr.aws/lts/ubuntu:latest
```

Comment:
So AmazonLinux is based on RHEL and mingw was removed from [EPEL](https://docs.fedoraproject.org/en-US/epel/) around 2 years back.
Reference: https://pagure.io/fesco/issue/2333

Therefore, as a workaround we are changing the base to Ubuntu. Now, changing the base for this image does not cause any issues since we are not using this image anywhere. It is only used to copy the build artifacts from inside the image to the final `aws-for-fluent-bit` image which is still based on AmazonLinux.
Reference: https://github.com/aws/aws-for-fluent-bit/blob/89a6da8e7c926342fba537c7325e632d8003d393/Dockerfile#L110

Reply:
Oh good call. This is just the builder image